apps/jubmoji-quest/src/pages/quests/[id].tsx

```diff
@@ -161,14 +157,24 @@ export default function QuestDetailPage() {
 
   const showLeaderboard = quest.proofType === $Enums.ProofType.TEAM_LEADERBOARD;
 
-  const collectionCardIndices = quest.collectionCards.map((card) => card.index);
-
-  const collectedItems =
-    jubmojis?.filter((jubmoji) =>
-      collectionCardIndices.includes(jubmoji.pubKeyIndex)
-    ).length ?? 0;
+  const collectionCardIndices = quest.powers
```

Comment:
So quest collection cards are not taken into account here?

Reply:
No